src/fu-engine.c

```diff
@@ -875,6 +875,38 @@ fu_engine_require_vercmp (XbNode *req, const gchar *version, GError **error)
 	return ret;
 }
 
+static gboolean
+fu_engine_check_requirement_not_child (FuEngine *self, XbNode *req,
```

Comment:
LVFS docs will need to get updated for this new requires extension

Reply:
I suppose if we're supporting it in the client then we ought to make the feature available to all vendors... I've opened https://github.com/hughsie/lvfs-website/issues/315 for when I have a brain for prose.